src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -699,7 +728,7 @@ machine(MachineType:TCC, "TCC Cache")
   // Stalling transitions do NOT check the tag array...and if they do,
   // they can cause a resource stall deadlock!
 
-  transition(WI, {RdBlk, WrVicBlk, Atomic, WrVicBlkBack}) { //TagArrayRead} {
+  transition(WI, {RdBlk, WrVicBlk, Atomic, AtomicPassOn, WrVicBlkBack}) { //TagArrayRead} {
```

Comment:
Do we want TagArrayRead for this?

Reply:
Removed as extraneous per @abmerop's comment.